src/components/services/PasswordReminderService.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import { isEqual } from 'lodash'
+import * as React from 'react'
+import { connect } from 'react-redux'
+
+import { setPasswordReminder } from '../../modules/UI/components/PasswordReminderModal/actions.js'
+import { type Dispatch, type RootState } from '../../types/reduxTypes.js'
+import type { PasswordReminder } from '../../types/types.js'
+
+type Props = {
+  setPasswordReminder: (passwordReminder: PasswordReminder) => void,
+  passwordReminder: PasswordReminder,
+  loginStatus: boolean | null
+}
+
+class PasswordReminderComponent extends React.PureComponent<Props> {
+  UNSAFE_componentWillReceiveProps(nextProps: Props) {
+    if (nextProps.loginStatus && !isEqual(this.props.passwordReminder, nextProps.passwordReminder)) {
```

Comment:
Do not use lodash routines. We are trying to deprecate it from the code base  Check the parameters manually